src/foam/u2/view/UnstyledTableView.js

```diff
@@ -706,6 +711,11 @@ foam.CLASS({
                       }).
                     end();
                   tbodyElement.add(tableRowElement);
+
+                  // reset actions back to base state
+                  actions = {
```

Comment:
Would it be simpler and easier to understand if we just had a method called getActionsForRow(obj) that created a new Action list for each row by merging contextMenuActions and the object's actions, rather than reusing and juggling this 'actions' var?

Reply:
I got too excited when I fixed the bug I'll add the method now

Reply:
Done.